__tools__/run-tests.sh

```diff
@@ -87,5 +114,80 @@ if [ $? == 0 ]; then
     die mock contract test succeeded though it should have failed
 fi
 
+## -----------------------------------------------------------------
+yell ---------- start pdo-create and pdo-update tests ----------
+## -----------------------------------------------------------------
+
+# make sure we have the necessary files in place
+CONFIG_FILE=${VIRTUAL_ENV}/opt/pdo/etc/pcontract.toml
```

Comment:
Shouldn't we be using CONTRACTHOME instead of assuming it's in the virtual environment under /opt/pdo? Sure, we recommend this configuration, but...